### PR TITLE
fixed bugs in decision integration test

### PR DESCRIPTION
**Pull request by ashum68**

### CODE REVIEW

File: tests/integration/test_decision_worker.py [hunk 1]
```diff
@@ -30,6 +31,7 @@ def simulate_data_merge_worker(in_queue: queue_wrapper.QueueWrapper) -> None:
     """
     Place example merged detection into the queue.
     """
+    time.sleep(2)
```
Comment (DanielJPuratich) [inline tests/integration/test_decision_worker.py]: Why is this needed? Does the worker take time to startup?

Comment (ashum68) [reply tests/integration/test_decision_worker.py]: Not needed. Thanks for catching that, updated.

**Review by TongguangZhang — APPROVED** [review]
approved
